src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -122,111 +122,113 @@
 Copyright (C) Microsoft Corporation. All rights reserved.</value>
   </data>
   <data name="ShellHelp" xml:space="preserve">
-    <value>PowerShell[.exe] [-Version] [-NoLogo] [-NoExit]
-    [-NoProfile] [-NonInteractive] [-Interactive]
-    [-InputFormat {Text | XML}] [-OutputFormat {Text | XML}]
-    [-WindowStyle &lt;style&gt;] [-EncodedCommand &lt;Base64EncodedCommand&gt;]
-    [-ConfigurationName &lt;string&gt;]
-    [-Command { - | &lt;script-block&gt; [-args &lt;arg-array&gt;]
-                  | &lt;string&gt; [&lt;CommandParameters&gt;] } ]
-    [-File &lt;filePath&gt; &lt;args&gt;] [-ExecutionPolicy &lt;ExecutionPolicy&gt;]
-
-PowerShell[.exe] -Help | -? | /?
-
--Version
-    Shows the version of PowerShell and exits.
-    Additional arguments are ignored.
-
--NoLogo
-    Hides the copyright banner at startup.
+    <value>PowerShell[.exe] [options]
 
--NoExit
-    Does not exit after running startup commands.
+PowerShell[.exe] -h | -Help | -? | /?
 
--NoProfile
-    Does not load the PowerShell profile.
+PowerShell Online Help https://docs.microsoft.com/en-us/powershell/scripting/powershell-scripting?view=powershell-6
```

Comment:
if this can't be a shorter URL then the URL should be on it's own line to attempt preventing line wrapping.

Reply:
I just created a short one that points to that you can use: https://aka.ms/pscore6-docs

Reply:
Fixed.